spring-cloud-gcp-samples/spring-cloud-gcp-kms-sample/README.adoc

```diff
@@ -0,0 +1,41 @@
+= Spring Cloud GCP KMS Sample Application
+
+This code sample demonstrates how to use the Spring Cloud GCP Key Management Service (KMS) integration.
+
+== Running the Sample
+
+1. Create a Google Cloud Platform project with https://cloud.google.com/billing/docs/how-to/modify-project#enable-billing[billing enabled], if you don't have one already.
+
+2. Enable the Cloud Key Management Service from the "APIs & Services" menu of the Google Cloud Console.
+This can be done using the `gcloud` command line tool:
+
+```
+gcloud services enable cloudkms.googleapis.com
+```
+
+3. Authenticate in one of two ways:
+
+a. Use the Google Cloud SDK to https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login[authenticate with application default credentials].
+This method is convenient but should only be used in local development.
+b. https://cloud.google.com/iam/docs/creating-managing-service-accounts[Create a new service account], download its private key and point the `spring.cloud.gcp.kms.credentials.location` property to it.
+Such as: `spring.cloud.gcp.credentials.location=file:/path/to/creds.json`
+
+4. Using the https://console.cloud.google.com/security/kms[Key Management Service in Cloud Console], create a new global key ring named `spring-cloud-gcp` to the location of your choice, and create a key on that key-ring.
```

Comment:
We can just ask them here to create a global key and tell them what to name the key, so that in Step 6a, we can tell them exactly what to paste into the box for the key.

Reply:
Changed it. They should create the global key `kms://spring-cloud-gcp/key-id`